lib/src/klio/transforms/io.py

```diff
@@ -109,14 +166,27 @@ def read_records(self, file_name, range_tracker):
             yield message.SerializeToString()
 
 
-class KlioReadFromText(beam.io.ReadFromText, _KlioTransformMixin):
+class _KlioReadFromText(beam.io.ReadFromText, _KlioTransformMixin):
     """Read from a local or GCS file with each new line as a
     ``KlioMessage.data.element``.
     """
 
     _source_class = _KlioReadFromTextSource
 
 
+class KlioReadFromText(beam.PTransform):
+    def __init__(self, *args, **kwargs):
+        self.reader = _KlioReadFromText(*args, **kwargs)
```

Comment:
I made these readers/writers public/semiprivate so they can be tested easier. I just realized that I'm not consistent re: making public or semiprivate. I can change that if y'all want. 

I didn't need to mock out the counter objects so I kept those private.